deploy-lvmo/deploy.sh

```diff
@@ -43,6 +43,12 @@ function extract_vars() {
     export CHANGEME_STORAGE_DEVICE_SET_COUNT="${disks_count}"
 }
 
+function extract_kubeconfig() {
```

Comment:
This one should come from `common.sh`

I'm assuming the execution failed for you, right? Let's try to see why it didn't get the function from `common.sh`

Reply:
Without it  the failure is:
[deploy-lvmo : deploy-lvmo] ./verify.sh: line 20: extract_kubeconfig: command not found

Reply:
then we should probably just call `extract_kubeconfig_common` instead. I think it does exactly what you need here :)

Reply:
Yeah, I started doing some of that a couple of PRs ago. Still lots to clean-up :)